### DISCUSSION TITLE
How to connect to a Postgresql server which requires a SSL Tunnel

### DISCUSSION THREAD

**Discussion by drowsytech** [apache/superset#14144] [Q&A / Help]
I have a postgresql server which is not accessible remotely, it can only be accessed using the SSH Tunnel. I am new to superset so not sure how to do it. I created a ssh tunnel redirect my local port to postgres server which connects successfully using the following command

ssh -N -L 5000:127.0.0.1:6432 192.168.1.55

After this when I tried to add database to superset it gives the error "Connection Failed please check your connection strings" I have tried so many combination but same error.

Any help with this is highly appreciated.

**Comment by JayAhn2** [comment]
Seems superset doesn't provide accessibility to DB which is not directly accessible. Have you manage to solve this problem?

**Comment by mohammeddib10** [comment]
@drowsytech Any news about this subject? I need to do the same thing.
